i3pystatus/pulseaudio/__init__.py

```diff
@@ -147,7 +147,7 @@ def update_cb(self, context, t, idx, userdata):
 
         self.request_update(context)
 
-    def sink_info_cb(self, context, sink_info_p, _, __):
+    def sink_info_cb(self, context, sink_info_p, eol, __):
```

Comment:
You can change this `__` to a single `_`.

Reply:
done